documentation/loginAndLogoutFlow.md

```diff
@@ -25,23 +25,20 @@ The reason for preferring the `/proximityTriggered` URL over the `/logout` and `
 
 The reason for continuing to support the specific /login and /logout, instead of only supporting the `/proximityTriggered` URL is that they make sense for some user listeners, for example when using the the USB listener, one wouldn't expect to log a user out when inserting the USB, or log a user in when pulling the USB out. So for that, it seems to make sense that it attempts to log in (i.e. call `/login`) or log out (i.e. call `/logout` when inserting/removing the USB, respectively
 
-
-
-
 ### Technical description
 
 The core part of the flow is defined in two files:
 
 * `UserLogonStateChange.js` contains the handling of the logon related endpoints kicks off the related process. It contains individual handlers for the `login`, `logout` and `proximityTriggered` URLs. These handlers all have the `gpii.flowManager.userLogonHandling.stateChangeHandler` grade, which is the component that contains the functionality for the actual logging in and logging off.
-* `FlowManagerUtitilities.js` which describes the remaining part of the flow (e.g. fetching resources, matchmaking, etc.).
+* `FlowManagerRequests.js` which describes the remaining part of the flow (e.g. fetching resources, matchmaking, etc.).
 
 The user login process is as follows:
 
 1. a GET request is sent to either `/user/:token/login` or `/user/:token/proximityTriggered`. This is retrieved by the relevant handler in `UserLogonStateChange`, and if it is found that the token needs to be logged in, the `onUserToken` event is fired (via the `gpii.flowManager.userLogonHandling.loginUser` function)
 1. the `onUserToken` event has three listeners:
    1. UserLogonStateChange's `getDeviceContext`, which fetches the device reporter data. When this has been fetched an `onDeviceContext` event is fired.
-   2. `getPreferences` (flowManagerUtilities), which fetches the preferences and fires the `onPreferences` event when the preferences are fetched.
-   3. `setUserToken` (flowManagerUtilities) which sets the userToken property in the handler
+   2. `getPreferences` (flowManagerRequests), which fetches the preferences and fires the `onPreferences` event when the preferences are fetched.
+   3. `setUserToken` (flowManagerRequests) which sets the userToken property in the handler
```

Comment:
If we use an initial capital "F" for `flowManagerRequests` in `cloudBasedFlow.md`, then here also